### PR TITLE
[Docs] Fix model monitoring setup Grafana instructions

### PR DESCRIPTION
**Pull request by katyakats**

### CODE REVIEW

File: docs/model_monitoring/model-monitoring-deployment.ipynb [hunk 1]
```diff
@@ -257,36 +257,13 @@
     "If you do not have a Grafana service running,\n",
     "see <a href=\"https://www.iguazio.com/docs/latest-release/services/fundamentals/#create-new-service\" target=\"_blank\">Creating a New Service</a> to create and configure it.\n",
     "\n",
-    "1. Make sure you have the `model-monitoring` as a Grafana data source configured in your Grafana service. If not,\n",
-    "add it by:\n",
+    "Add access keys to your model-monitoring data source by:\n
```
Comment (jasonnIguazio) [inline docs/model_monitoring/model-monitoring-deployment.ipynb:260]: Correct to: "Add access keys to your model-monitoring data source:\n

File: docs/model_monitoring/model-monitoring-deployment.ipynb [hunk 2]
```diff
@@ -257,36 +257,13 @@
...
-    "    * {download}`Model Monitoring - Performance <./dashboards/model-monitoring-performance.json>`\n",
-    "\n",
-    "3. Import the downloaded dashboards to your Grafana service.\n",
-    "To import that dashboards into your Grafana service:\n",
-    "    1. Navigate to your Grafana service in the Services list and press on it\n",
-    "    2. Press the dashboards icon in left menu\n",
-    "    3. In the dashboard management screen press the IMPORT button, and select one file to import. Repeat this step for each dashboard.\n",
-    "\n",
+    "   3. Press `mlrun-model-monitoring`.\n",
+    "   4. In Custom HTTP Headers configure the cookie parameter. Value should be `session=j:{\"sid\": \"<YOUR ACCESS KEY>\"}`\n",
```
Comment (jasonnIguazio) [inline docs/model_monitoring/model-monitoring-deployment.ipynb:264]: Correct to: In Custom HTTP Headers, configure the cookie parameter. Set the value to `session=j:{\"sid\": \"<YOUR ACCESS KEY>\"}`\n",

File: docs/model_monitoring/model-monitoring-deployment.ipynb [hunk 3]
```diff
@@ -257,36 +257,13 @@
...
     "```{note}\n",
+    "You will find overview, details and performance dashboards in `Dashboards -> Manage -> private`"
```
Comment (jasonnIguazio) [inline docs/model_monitoring/model-monitoring-deployment.ipynb:266]: Correct to: "You will find overview, details, and performance dashboards in `Dashboards -> Manage -> private`"

**Review by jasonnIguazio — COMMENTED** [review]
I am pretty sure I got all the "+" that were changed in the file. If there was something I missed let me know.

**Review by jasonnIguazio — APPROVED** [review]

**Review by Hedingber — APPROVED** [review]

**Pull request merged**